packages/common/src/utils/message.js

```diff
@@ -95,6 +96,9 @@ const isNodeEqual = (n1, n2) => {
     case "image-attachment":
       return propertiesEqual(["url"]);
 
+    case "horizontal-divider":
+      return n1.type === n2.type;
```

Comment:
Ah is this to avoid duplicate dividers?

Reply:
hm no 🤔

Reply:
Without this it’d fall back to the default comparison function (`baseEqual`), which would check that the children are equal. It’d work, it’s just unnecessary for dividers.

Reply:
(Plan to move these into each individual plugin file)